crates/core/tedge_agent/Cargo.toml

```diff
@@ -29,9 +29,13 @@ anyhow = "1.0"
 async-trait = "0.1"
 clap = { version = "3.2", features = ["cargo", "derive"] }
 flockfile = { path = "../../common/flockfile" }
+futures-util = "0.3"
+hyper = { version = "0.14", features = ["full"] }
+lazy_static = "1.4"
 futures = "0.3"
 mqtt_channel = { path = "../../common/mqtt_channel" }
 plugin_sm = { path = "../plugin_sm" }
+routerify = "3.0"
```

Comment:
Why do we need `routerify`? We only need a handler per method.

Reply:
It was the only library I found that allowed for glob pattern route paths (`/users/*`) below:

```rust
use routerify::Router;
use hyper::{Response, Body};

let router = Router::builder()
    .get("/users/*", |req| async move { Ok(Response::new(Body::from("It will match /users/, /users/any_path"))) })
    .build()
    .unwrap();
```